sphinx-ecosystem-news-january-2022.rst

```diff
@@ -0,0 +1,115 @@
+   which resulted in user frustration.
+   This problem has now been fixed and autosummary properly points out the reason of the ``ImportError``.
+
+You can read the `complete Sphinx 4.4.0 release
+notes <https://www.sphinx-doc.org/en/master/changes.html#release-4-4-0-released-jan-17-2022>`_ online.
+
+**pydata-sphinx-theme**, a Sphinx theme based on Bootstrap CSS developed by the PyData community,
+`got a new 0.8 release <https://github.com/pydata/pydata-sphinx-theme/releases/tag/v0.8.0>`_
+on January 15th. The Read the Docs team collaborated with the developers
+to make some of the new features behave correctly in our platform, in particular
+`the custom version
+switcher <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#add-a-dropdown-to-switch-between-docs-versions>`_.
+As `Chris Holdgraf summarizes in this Twitter
+thread <https://twitter.com/choldgraf/status/1482435411301449729>`_,
+the new version brings other interesting additions,
+like `better depth control of the left
+sidebar <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#navigation-depth-and-collapsing-of-the-sidebar>`_
+and `custom SVG
+icons <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#local-image-icons>`_.
+
+Extensions
+----------
+
+Several new extensions saw new releases recently, including:
+
+`sphinx-codeautolink 0.9 <https://sphinx-codeautolink.readthedocs.io/en/stable/release_notes.html#id2>`_
+   sphinx-codeautolink is a Sphinx extension to automatically include links inside code blocks.
+   The 0.9 version now links Python builtin objects if intersphinx is properly configured
+   and has several logging improvements.
+ 
+   .. figure:: /img/sphinx-codeautolink.gif
+      :align: center
+      :alt: Demo of sphinx-codeautolink
+ 
+      Demo of sphinx-codeautolink
+
+`sphinxcontrib-constdata 1.0 <https://documatt.gitlab.io/sphinxcontrib-constdata/>`_
+   sphinxcontrib-constadata is a new Sphinx extension that allows documentation writers to
+   easily load data stored in CSV, JSON, and YAML files.
+   For example, it can be used to load UI labels (button labels, menu selection labels)
+   from external files instead of hardcoding them in the documentation for better maintenance,
+   for example:
+ 
+   .. code-block:: rst
+ 
+      Choose menu item :constdata:label:`menu.yaml?FileSaveAs`.
+
+Upcoming
+--------
+
+The Executable Books Project team is working on several exciting things around MyST,
+including `the upcoming MyST-Parser 0.17 release <https://github.com/executablebooks/MyST-Parser/pull/507>`_
+and `direct integration with Jupyter notebooks <https://twitter.com/choldgraf/status/1485666900784730112>`_.
+
+.. figure:: /img/jupyter-myst.gif
+   :align: center
+   :alt: Preview of MyST integrated in Jupyter notebooks.
+
+   Preview of MyST integrated in Jupyter notebooks.
```

Comment:
This is an abrupt ending... We should close this out with something.

Reply:
Fair enough 😅